app/src/main/java/com/apps/adrcotfas/goodtime/Preferences.java

```diff
@@ -75,4 +89,28 @@ public String getNotificationSound() {
     public void setDisableSoundAndVibration(boolean disableSoundAndVibration) {
         mPref.edit().putBoolean(DISABLE_SOUND_AND_VIBRATION, disableSoundAndVibration).apply();
     }
+
+    public void migrateFromOldPreferences(SharedPreferences oldPref) {
+        if(oldPref.getBoolean(SESSION_DURATION, true)) {
```

Comment:
@fdw
This line causes a crash after update and it's on Google Play.
I'm trying to fix it now. Do you know what is the problem here?